docs/configuration/configuration.rst

```diff
@@ -2,5 +2,168 @@
+The rest of the section can be used to configure uniform abundances for all shells, by giving the atom name and a
+relative abundance fraction. If it does not add up to 1., TARDIS will warn - but normalize the numbers.
+
+
+
+Plasma
+------
+
+.. jsonschema:: schemas/plasma.yml
+
+``inital_t_inner`` is temperature of the black-body on the inner boundary. ``initial_t_rad`` is the
+radiation temperature for all cells. For debugging purposes and to compare to :term:`synapps` calculations one can
+disable the electron scattering. TARDIS will issue a warning that this is not physical.
+There are currently two ``plasma_type`` options available: ``nebular`` and ``lte`` which tell TARDIS how to run the
+ionization equilibrium and level population calculations (see :ref:`plasmas` for more information).
+The radiative rates describe how to calculate the :math:`J_\textrm{blue}` needed for the :ref:`nlte` calculations and
+:ref:`macroatom` calculations. There are three options for ``radiative_rates_type``: 1) ``lte`` in which
+:math:`J_\textrm{blue} = \textrm{Blackbody}(T_\textrm{rad})`, 2) ``nebular`` in which
+:math:`J_\textrm{blue} = W \times \textrm{Blackbody}(T_\textrm{rad})`, 3) ``detailed`` in which the :math:`J_\textrm{blue}`
+are calculated using an estimator (this is described in ??????).
+
+TARDIS currently supports three different kinds of line interaction: ``scatter`` - a resonance scattering implementation,
+``macroatom`` - the most complex form of line interaction described in :ref:`macroatom` and ``downbranch`` a simplified
+version of ``macroatom`` in which only downward transitions are allowed.
+
+Finally, ``w_epsilon`` describes the dilution factor to use to calculate :math:`J_\textrm{blue}` that are 0, which
+causes problem with the code (so :math:`J_\textrm{blue}` are set to a very small number).
+
+NLTE
+^^^^
+
+.. code-block:: yaml
+
+    nlte:
+        coronal_approximation: True
+        classical_nebular: False
+
+The NLTE configuration currently allows setting ``coronal_approximation`` which sets all :math:`J_\textrm{blue}` to 0.
+This is useful for debugging with :term:`chianti` for example. Furthermore one can enable 'classical_nebular' to set all
+:math:`\beta_\textrm{Sobolev}` to 1. Both options are used for checking with other codes and should not be enabled in
+normal operations.
+
+
+
+Monte Carlo
+-----------
+
+The ``montecarlo`` section describes the parameters for the MonteCarlo radiation transport and convergence criteria:
+
+.. jsonschema:: schemas/montecarlo.yml
+
+The ``seed`` parameter seeds the random number generator first for the creation of the packets
+(:math:`\nu` and :math:`\mu`) and then the interactions in the actual MonteCarlo process.
+The ``no_of_packets`` parameter can take a float number for input convenience and gives the number of packets normally
+used in each MonteCarlo loop. The parameters ``last_no_of_packets`` and ``no_of_virtual_packets`` influence the last run
```

Comment:
specify that you refer to the MC cycles in which the plasma state is iteratively found